### ISSUE TITLE
Android app missing from google play

### ISSUE THREAD

**Issue by alexbozhenko** [berty/berty#4872]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Berty product

Mobile app

### Berty product version

0

### OS

Android

### OS version

0

### Device

0

### Steps to reproduce

1. Go to https://play.google.com/store/apps/details?id=tech.berty.android
2. Link is broken

### Current behavior

Can not download app

### Expected behavior

Should be able to download...

### Other

Is this related to https://github.com/berty/berty/issues/4851 ?

**Comment by D4ryl00** [comment]
Hello, thanks for opening this issue.
We are currently working on getting back our Google Play developer account.
We will let you know here when we will know more about it.

**Comment by alexbozhenko** [comment]
@D4ryl00 thank you so much for the quick response!
Have you considered putting it on https://f-droid.org/ ?

**Comment by D4ryl00** [comment]
Yes for sure, but for now, we are still working on maintaining the app, until we have more resources to allocate to Berty Messenger.
https://github.com/berty/berty/issues/3101#issuecomment-1551484180

**Comment by Faolain** [comment]
Can this ar least have an apk built in the meantime?

**Comment by D4ryl00** [comment]
Unfortunately the Android build is broken because of very old dependencies that cannot be downloaded anymore. We are working on upgrading the whole front-end project (Expo -> React-Native -> dependencies).

**Comment by jefft0** [comment]
We'll keep this open as a tracking issue for updating the Play account access and publishing the new Android version.

**Comment by jefft0** [comment]
@D4ryl00 will ask "administrator" for documents needed for Google Play.

**Comment by SohamAnd** [comment]
i would like to contribute in this project and to solve this issue.

**Comment by jefft0** [comment]
@alexbozhenko, we just released a new version on Google Play Store. Can you try it?
https://play.google.com/store/apps/details?id=tech.berty.android

<blockquote><img src="https://play-lh.googleusercontent.com/JhUTgA5Mr63loQJQu3DGrY9oMPLTN7kHQvP378fuHNB89fPotDSrX80PVVPJadlqOMJIKtgxU0ZqOrWrHrbJ" width="48" align="right"><div><strong><a href="https://play.google.com/store/apps/details?id=tech.berty.android&hl=en_US">Berty Messenger - Apps on Google Play</a></strong></div><div>You deserve privacy!</div></blockquote>

**Comment by jefft0** [comment]
Closing as assumed resolved.
